index.html

```diff
@@ -194,17 +214,108 @@ <h4>Frameworks</h4>
            <li><div class="skills">
                 <h4>Skills</h4>
                 
-                <ul>
+                <ul class="disc">
                     <li><p>Codeit</p></li>
                     <li><p>Codepen</p></li>
                     <li><p>GitHub</p></li>
                     <li><p>Terminal</p></li>
                 </ul>
             </div></li>
+            <li><div class="yeah"></div></li>
         </ul>
+
+        <div class="portion">
+            <div class="aboutword"><h1>About</h1></div>
+            <div class="abouttext"><p>Lorem ipsum 
+                dolor sit amet, consectetur 
+                adipiscing elit, sed do eiusmod 
+                tempor incididunt ut labore et 
+                dolore magna aliqua. Ut enim ad 
+                minim veniam, quis nostrud 
+                exercitation ullamco
+                 laboris nisi ut aliquip ea eiusmod</p>
+                </div>
             
-        
+                <div class="getflex">
+                    <a href="#" alt="#"><div class="cvbox"><p>Get my CV</p></a>
+            <div class="union-1"><img src="assets/ic_download.svg" class="arrow-down" alt="#"></div></div>
+        </div>
+            
+       
     </section>
 </section>
+
+<footer>
+        <div class="footspace"></div> 
+        <div class=" box namet"><h3>Get Started</h3></div>
+        <div class=" box introt"><p>Lorem ipsum dolor sit amet,
+             consectetur adipiscing elit, sed do eiusmod tempor
+              incididunt ut labore et dolore magna.</p></div>
+<form  action="https://formspree.io/f/xknyorrb" method="post">
+   <div class="fooc namei">
+        <fieldset>
+            <div class="field">
+            <legend>Your name</legend>
+            <input type="text" name="username" maxlength="30" placeholder="Last name First name" required>
+        </div>
+        </fieldset>
+    </div>
+    <div class="fooc email">
+        <fieldset>
+            <div class="field">
+            <legend>Your email</legend>
+            <input type="text" name="email" placeholder="someone@domain.com" required>
+        </div>
+        </fieldset>
+    </div>
+    <div class="fooc textmessage">
+        <fieldset>
+            <div class="field">
+                <textarea name="comment" maxlength="500" placeholder="Enter your message here..." required></textarea>
+            </div>
+        </fieldset>
+    </div>
+        <button value="submit" class="collabo">
+            <div>Start collaboration</div></button> 
```

Comment:
Please note that it is semantically incorrect to put a `<div>` inside of a `<button>`. I also observed that you have some `a` tags with `<div>` inside of them. Please note that this is not correct semantically.